.github/workflows/integration.yml

```diff
@@ -9,7 +9,57 @@ on:
     branches: [ master ]
 
 jobs:
+  Canal-integration:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.14.0'
+
+      - name: Cache Vendor
+        id: cache-vendor
+        uses: actions/cache@v2
+        with:
+          path: vendor
+          key: ${{ runner.os }}-cdc-integration-vendor-${{ hashFiles('go.sum') }}
+
+      - name: Update Vendor
+        run: go mod vendor
+
+      - name: Pull images
+        run: docker-compose -f docker-compose-canal.yml pull --ignore-pull-failures
```

Comment:
`--ignore-pull-failures`
Curious about what happens when error happens during image pulling

Reply:
In our situation, running this command without `--ignore-pull-failures` will **always** fail because it will try to pull all images from DockerHub, even if it's a custom image and should be built locally.